src/graphql/queries/__tests__/ListArticles.js

```diff
@@ -1010,5 +1010,30 @@ describe('ListArticles', () => {
     `);
   });
 
+  it('lists all articles with cooccurrences', async () => {
+    expect(
+      await gql`
+        {
+          ListArticles {
```

Comment:
nit: we can limit the liststed article ids to `listArticleTest1`, `listArticleTest2` (those with cooccurrences) and `listArticleTest3` (expected to have empty cooccurrences) using `ids` filter in `ListArticleFilter` to avoid unnecessary snapshot changes when we add new fixtures in the future.

Reply:
updated